src/model/reminder.ts

```diff
@@ -20,7 +20,7 @@ export class Reminder {
     public title: string,
     public time: DateTime,
     public rowNumber: number,
-    public done: boolean
+    public status: string,
```

Comment:
How about changing this to the `ReminderStatus` enum?

I feel changing the type of `status` to `ReminderStatus` improves type safety, code readability, and maintainability by ensuring valid values, clearly indicating allowed statuses, and simplifying future updates.